runtime/queries/thrift/indents.scm

```diff
@@ -0,0 +1,9 @@
+(definition) @indent
+
+((parameters (parameter)) @aligned)
+
+"}" @outdent
+
+[ "{" "}" ] @branch
+
+[ "(" ")" ] @branch
```

Comment:
We don't use `@branch` in our indent queries (https://docs.helix-editor.com/master/guides/indent.html#capture-types). For now we could remove this file and rely on the default indent behavior. Indent queries can always be contributed as a follow-up